.github/workflows/scripts/check-jira-issues.sh

```diff
@@ -8,42 +8,53 @@ set -euo pipefail
 PROJECTS="$1"
 RELEASE="$2"
 PATCH="$3"
-RELEASE_PATCH="$4"
+
+RELEASE_PATCH="${RELEASE}.${PATCH}"
 
 check_not_empty \
     PROJECTS \
     RELEASE \
     PATCH \
-    RELEASE_PATCH \
     \
     JIRA_TOKEN \
     DRY_RUN
 
 # TODO: Jira returns 400 if requested fixVersion does not exist. That means
 # the named release must exist on Jira, which is not given.
 JQL_OPEN_ISSUES="project IN ($PROJECTS) \
-AND fixVersion = \"$RELEASE.$PATCH\" \
+AND fixVersion = \"$RELEASE_PATCH\" \
 AND statusCategory != done \
-AND (Component IS EMPTY or Component NOT IN (Documentation, \"ACS Managed Service\")) \
+AND (Component IS EMPTY or Component NOT IN (Documentation, \"ACS Cloud Service\")) \
 AND type NOT IN (Epic, \"Feature Request\") \
 ORDER BY assignee"
 
 JQL_CLOSED_WITH_PR="project IN ($PROJECTS) \
-AND fixVersion = \"$RELEASE.$PATCH\" \
+AND fixVersion = \"$RELEASE_PATCH\" \
 AND statusCategory = done \
-AND (Component IS EMPTY or Component NOT IN (Documentation, \"ACS Managed Service\")) \
+AND (Component IS EMPTY or Component NOT IN (Documentation, \"ACS Cloud Service\")) \
 AND issue.property[development].openprs > 0 \
 ORDER BY assignee"
 
 get_issues() {
+    local status=0
     curl --fail -sSL --get --data-urlencode "jql=$1" \
         -H "Authorization: Bearer $JIRA_TOKEN" \
         -H "Accept: application/json" \
-        "https://issues.redhat.com/rest/api/2/search"
+        "https://issues.redhat.com/rest/api/2/search" || status="$?"
+    if [[ "$status" == 22 ]]; then  # curl man page: "22     HTTP  page  not retrieved. .. HTTP error code being 400 or above."
+        gh_log warning "4XX http error code" >&2
```

Comment:
This would be masking the failure to find the version on Jira as it is not hard-stopping the pipeline (think release engineer has a typo and cut-rc for 3.89 instead of 3.78).